.github/workflows/target-determination-indexer.yml

```diff
@@ -0,0 +1,114 @@
+name: Index PyTorch Tests for Target Determination
+
+on:
+  workflow_dispatch:
+  # TODO: Trigger every few hours
```

Comment:
Could probably use [`schedule`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) for simplicity?

Reply:
Yes this is the plan - not going to turn on this workflow just yet so leaving the TODO in here